sinatra-contrib/Gemfile

```
@@ -16,8 +16,7 @@ group :development, :test do
   end
 
   platform :jruby, :ruby do
-    gem 'hamlit'
-    gem 'hamlit-block', '>= 0.7.1'
+    gem 'hamlit', '~> 3'
```

IIRC, `~> 3.0.0` allows any 3.0.x and `~> 3.0` allows any 3.x. So... ~> 3 would allow any x. Maybe just writing `>= 3` is simpler? Just to be clear, I don't want us to be locked to 3.x either.
```suggestion
    gem 'hamlit', '>= 3'
```